manifests/charts/ambient/values.yaml

```diff
@@ -0,0 +1,32 @@
+# Copied from `helm-profiles/ambient.yaml`
+
+global:
+  variant: distroless
+
+# Overrides for the `istiod-remote` dep
+istiod:
```

Comment:
I think helm can do some enablement things. Do we want that here?

base, cni, and ztunnel are ~singletons so having them together is fairly reasonable. Prod usage of istiod should use revisions though, so it seems like you would want to actually have `ambient` and `istiod-<rev>` for most prod usage?

Unless we are claiming this is just for simple use cases and prod should always fully separate?

Reply:
You could do that with `helm install ambient --set istiod.revision`

Reply:
doesn't that remove the old one instantly though

Reply:
it shouldn't, but lemme check.

Reply:
If not, what is managed the old revision?

Reply:
don't we need a `enabled` condition in chart.yaml then to allow to later remove the Istiod original

Reply:
Yeah good call - I can add a dependency conditional.

Reply:
tbh might be good to just pre-emptively do it for all of them. But we can also do it on-demand if someone asks

Reply:
Yeah latter preferred, since none of the others support revisions. 

I would kinda want to have eyeballs on decisions for making the other two conditional for $reason, so prefer to defer-as-requested.